packages/account/src/configs.test.ts

```diff
@@ -7,6 +7,11 @@ describe('Configs', () => {
     expect(configs.FUEL_NETWORK_URL).toBe('http://127.0.0.1:4000/v1/graphql');
   });
 
+  it('exports FUEL_DEVNET_NETWORK_URL', async () => {
+    const configs = await import('./configs');
+    expect(configs.FUEL_DEVNET_NETWORK_URL).toBe('https://devnet.fuel.network/v1/graphql');
```

Comment:
That's great!

Can we trim these constant names a bit?

 - `DEVNET_NETWORK_URL`
 - `TESTNET_NETWORK_URL`